src/components/EmailAuth.vue

```diff
@@ -289,7 +280,40 @@ export default {
     },
     backToSocial () {
       this.$emit('update:emailPass', false)
+    },
+    switchLogin () {
+      this.login = !this.login
+      this.resetPassword = false
+    },
+    sendPasswordResetEmail () {
+      this.$v.$touch()
+      if (this.$v.invalid) {
+        this.submitStatus = 'ERROR'
+      } else {
+        this.submitStatus = 'PENDING'
+        auth.sendPasswordResetEmail(this.formData.email).then((result) => {
+          this.$store.commit('notification/push', {
+            message: this.$t('login.reset-password-email-sent'),
+            title: this.$t('global.success'),
+            type: 'success'
+          }, { root: true })
+          this.resetPassword = false
+        }).catch((err) => {
+          this.$store.commit('notification/push', {
+            message: err.message,
```

Comment:
we should translate auth error messages: https://firebase.google.com/docs/reference/js/firebase.auth.Auth#sendpasswordresetemail